test/Microsoft.ML.OnnxTransformTest/OnnxTransformTests.cs

```diff
@@ -0,0 +1,215 @@
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.ImageAnalytics;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.RunTests;
+using Microsoft.ML.Runtime.Tools;
+using Microsoft.ML.Transforms;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using System.Runtime.InteropServices;
+using Microsoft.ML.OnnxScoring;
+
+namespace Microsoft.ML.Tests
+{
+    public class OnnxTransformTests : TestDataPipeBase
+    {
+
+        private const int inputsize = 150528;
+
+        private class TestData
+        {
+            [VectorType(inputsize)]
+            public float[] data_0;
+        }
+        private class TestDataSize
+        {
+            [VectorType(2)]
+            public float[] data_0;
+        }
+        private class TestDataXY
+        {
+            [VectorType(inputsize)]
+            public float[] A;
+        }
+        private class TestDataDifferntType
+        {
+            [VectorType(inputsize)]
+            public string[] data_0;
+        }
+
+        private float[] getSampleArrayData()
+        {
+            var samplevector = new float[inputsize];
+            for (int i = 0; i < inputsize; i++)
+                samplevector[i] = (i / (inputsize * 1.01f));
+            return samplevector;
+        }
+
+        public OnnxTransformTests(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [Fact]
+        void TestSimpleCase()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+                return;
+            var modelFile = "squeezenet/00000001/model.onnx";
+
+            var samplevector = getSampleArrayData();
+
+            var dataView = ComponentCreation.CreateDataView(Env,
+                new List<TestData>(new TestData[] {
```

Comment:
>new List<TestData>(new TestData[] [](start = 16, length = 33)

choose one.

Reply:
can you clarify? we have two examples to score -- did you mean to say just score 1 example? 

---
In reply to: [220398450](https://github.com/dotnet/machinelearning/pull/942#discussion_r220398450) [](ancestors = 220398450)

Reply:
You create array, and then pass it into List initializer. Why? CreateDataView can pick array, you don't need to wrap it into List. 

---
In reply to: [220408076](https://github.com/dotnet/machinelearning/pull/942#discussion_r220408076) [](ancestors = 220408076,220398450)

Reply:
I've updated to use only array.